mmv1/products/cgc/terraform.yaml

```diff
@@ -51,6 +51,23 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           deletion_protection: "false"
         ignore_read_extra:
           - "deletion_protection"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_mysql_instance_backup"
+        primary_resource_id: "instance"
```

Comment:
snippets for handwritten resources (everything in this file) will need the extra line for `primary_resource_type`. In these examples:
`primary_resource_type: "google_sql_database_instance"`